src/platform/mbed/net/include/BSDSocket.h

```diff
@@ -20,11 +20,13 @@
 
 #include "OpenFileHandleAsFileDescriptor.h"
 #include "common.h"
+#include "mbed-trace/mbed_trace.h"
 #include <atomic>
 #include <mstd_atomic>
 #include <netsocket/TCPSocket.h>
 #include <netsocket/UDPSocket.h>
 
+#define TRACE_GROUP "BSDS"
```

Comment:
That's not correct to put traces in headers. I suggest moving them into a cpp file.

Reply:
We do not have the cpp file for BSDSocket, but I guess in this case it should be added. I will change this